addons/api/fnc_addRackToVehicle.sqf

```diff
@@ -85,16 +85,31 @@ private _allowed = [_vehicle, _allowed] call EFUNC(sys_core,processVehicleSystem
 private _disabled = [_vehicle, _disabled] call EFUNC(sys_core,processVehicleSystemAccessArray);
 _intercoms = _intercoms apply {toLower _x};
 
-// A player must do the action of adding a rack
-private _player = objNull;
-
-if (isDedicated) then {
-    // Pick the first player
-    _player = (allPlayers - entities "HeadlessClient_F") select 0;
-} else {
-    _player = acre_player;
-};
+[{
+    // A player must do the action of adding a rack
+    private _player = objNull;
```

Comment:
Doubled code in condition and statement, put into function (can be local function above).

Reply:
why is the condition !isNil "_player" that is the same as writing true? did you maybe mean !isNull _player?

Reply:
if the player is not fully initialised it is nil. That is what testing showed